backend/app/migrations/0006_initial_data.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 3.1.5 on 2021-06-29 19:31
+
+from django.db import migrations
+
+
+def add_objects(apps, schema_editor):
+    pronounseries = apps.get_model('app', 'PronounSeries')
+    gender = apps.get_model('app', 'Gender')
+
+    he_series = pronounseries.objects.create(identifier = "Masc",subj = "he",obj = "him",pos_det = "his",pos_pro = "his",reflex = "himself")
+    Male = gender.objects.create(label = "Male")
+    Male.pronoun_series.add(he_series)
+
+    she_series = pronounseries.objects.create(identifier = "Fem",subj = "she",obj = "her",pos_det = "hers",pos_pro = "her",reflex = "herself")
+    Female = gender.objects.create(label = "Female")
+    Female.pronoun_series.add(she_series)
+
+    they_series = pronounseries.objects.create(identifier = "Andy",subj = "they",obj = "them",pos_det = "theirs",pos_pro = "their",reflex = "themself")
+    Nonbinary = gender.objects.create(label = "Nonbinary")
+    Nonbinary.pronoun_series.add(they_series)
+
+    xe_series = pronounseries.objects.create(identifier = "Xe",subj = "xe",obj = "xem",pos_det = "xyr",pos_pro = "xyrs",reflex = "xemself")
+    ae_series = pronounseries.objects.create(identifier = "Ae",subj = "ae",obj = "aer",pos_det = "aers",pos_pro = "aers",reflex = "aerself")
+    fae_series = pronounseries.objects.create(identifier = "Fae",subj = "fae",obj = "faer",pos_det = "faer",pos_pro = "faers",reflex = "faerself")
+    ey_series = pronounseries.objects.create(identifier = "Ey",subj = "ey",obj = "em",pos_det = "eir",pos_pro = "eirs",reflex = "eirself")
+    ve_series = pronounseries.objects.create(identifier = "Ve",subj = "ve",obj = "ver",pos_det = "vis",pos_pro = "vis",reflex = "verself")
+    per_series = pronounseries.objects.create(identifier = "Per",subj = "per",obj = "per",pos_det = "per",pos_pro = "pers",reflex = "perself")
+    ze_hir_series = pronounseries.objects.create(identifier = "Ze",subj = "ze",obj = "hir",pos_det = "hir",pos_pro = "hirs",reflex = "hirself")
+
+    all_series = [xe_series,ae_series,fae_series,ey_series,ve_series,per_series,ze_hir_series]
+    Neogenders = gender.objects.create(label = "Neo")
+    for each_series in all_series:
+        Neogenders.pronoun_series.add(each_series)
```

Comment:
This looks great overall! My one comment is that the capital-case variable name is a convention for classes (i.e., `Gender`, `PronounSeries`, etc.), not instances (i.e., `new_pronoun_series = PronounSeries.create(...)`). In lines 7 and 8 above, for instance, to follow the convention would be to write:
```
    PronounSeries = apps.get_model('app', 'PronounSeries')
    Gender = apps.get_model('app', 'Gender')
```

And elsewhere, to use lower case to reference class instances. ex:
```
female = gender.objects.create(label = "Female")
female.pronoun_series.add(she_series)
```

I'd recommend following the convention here, if possible.

Reply:
This makes sense. I have gone back and followed that convention in the migrations file.